doc/memos/rdm0.md

```diff
@@ -0,0 +1,153 @@
+
```

Comment:
Something to consider: maybe use `rdm0000.md` as a filename. That would make sorting easier beyond 10 RDMs ;-).

Reply:
(I'm only talking filename here; I don't see any benefit in making that also mandatory for e.g. "RDM 0" when used in the text below).